.github/composite_actions/log_metric/action.yaml

```diff
@@ -0,0 +1,52 @@
+name: 'Log Metric'
+description: 'Log data point to a metric with the provided value. If the metric is not there, it will create one.'
+inputs:
```

Comment:
request: on a high level, this should support some way for additional dimensions on the metric. At the very least, this could be hardcoded in this PR for flutter channel and version (as the canaries test across this metric) and the sent metrics should include these as dimensions so we can make the alarms on specific channels/versions if desired, or aggregate them. As written, the alarm will include all the data with no way to separate it later.

Reply:
Update dimensions data as agreed on yesterday.